docs/user-guide/filtering-layers.md

```diff
@@ -61,6 +61,14 @@ Also this filter is applied through the [Query Panel](filtering-layers.md#query-
 
 ### Quick Filter
 
+The user can performance two type of quick filters:
+
+* Filter by **attributes** 
+
+* Filter by **geometries**
+
+#### Quick Filter by attributes
+
 This filter, applicable directly in the [Attribute Table](attributes-table.md) just below the field names, can be also used in combination with other filter applied:
```

Comment:
This filter is available for each colum in the [Attribute Table](attributes-table.md) just below the field names and it can be also used in combination with other filter applied: